### ISSUE TITLE
Refresh API Documentation

### ISSUE THREAD

**Issue by g-back** [abapGit/abapGit#4487]
[API Documentation](https://github.com/abapGit/abapGit/blob/master/docs/collections/_development/api.md) could use a refresher, for example call
```abap 
DATA(lt_list) = NEW zcl_abapgit_persistence_repo( )->list( ).
``` 
is not valid anymore. There may be other deprecated calls too.

**Comment by larshp** [comment]
thanks, pull request welcome 😄

**Comment by mbtools** [comment]
it actually needs something like `zcl_abapgit_api` but then it becomes too "official"

**Comment by larshp** [comment]
only thing constant is change 😅

**Comment by nesl9** [comment]
I also ran into this issue. This worked for me:

DATA(lt_list) = zcl_abapgit_persist_factory=>get_repo( )->list( ).

**Comment by nesl9** [comment]
I've been using the API documentation recently. I can submit PRs as I go along and find issues.

**Comment by g-back** [comment]
closed with #4535? @larshp
